app/views/helpers/summary/Card7ForTransport.scala

```diff
@@ -0,0 +1,241 @@
+    if (transport.meansOfTransportCrossingTheBorderType.isDefined && transport.meansOfTransportCrossingTheBorderIDNumber.isDefined) {
+
+      val messagesForBorderMeansOfTransport = {
+        (transport.meansOfTransportCrossingTheBorderType, transport.meansOfTransportCrossingTheBorderIDNumber) match {
+          case (Some(meansType), Some(meansId)) => Seq(messages(s"declaration.summary.transport.border.meansOfTransport.$meansType"), meansId)
+          case _                                => Seq.empty
+        }
+      }
+
+      Some(
+        SummaryListRow(
+          key("transport.border.meansOfTransport.header"),
+          valueHtml(messagesForBorderMeansOfTransport.mkString("<br>")),
+          classes = "activeTransportType",
+          changeLink(BorderTransportController.displayPage, "transport.border.meansOfTransport.header", actionsEnabled)
+        )
+      )
+    } else None
+
+  private def transportPayment(transport: Transport, actionsEnabled: Boolean)(implicit messages: Messages): Option[SummaryListRow] =
+    transport.transportPayment.map { transportPayment =>
+      SummaryListRow(
+        key("transport.payment"),
+        valueKey(s"declaration.summary.transport.payment.${transportPayment.paymentMethod}"),
+        classes = "transportPayment",
+        changeLink(TransportPaymentController.displayPage, "transport.payment", actionsEnabled)
+      )
+    }
+
+  private def transportCrossingTheBorder(transport: Transport, actionsEnabled: Boolean)(implicit messages: Messages): Option[SummaryListRow] =
+    transport.transportCrossingTheBorderNationality.map { transportCrossingTheBorderNationality =>
+      SummaryListRow(
+        key("transport.registrationCountry"),
+        value(transportCrossingTheBorderNationality.countryName.getOrElse(messages("declaration.summary.unknown"))),
+        classes = "activeTransportCountry",
+        changeLink(TransportCountryController.displayPage, "transport.registrationCountry", actionsEnabled)
+      )
+    }
+
+  private def expressConsignment(transport: Transport, actionsEnabled: Boolean)(implicit messages: Messages): Option[SummaryListRow] =
+    transport.expressConsignment.map { expressConsignment =>
+      SummaryListRow(
+        key("transport.expressConsignment"),
+        valueKey(if (expressConsignment.isYes) "site.yes" else "site.no"),
+        classes = "expressConsignment",
+        changeLink(ExpressConsignmentController.displayPage, "transport.expressConsignment", actionsEnabled)
+      )
+    }
+
+  private def containers(transport: Transport, actionsEnabled: Boolean)(implicit messages: Messages): Option[List[SummaryListRow]] =
+    transport.containers flatMap { containers =>
+      val rows = containers.flatMap(containerRows(_, actionsEnabled))
+
+      if (rows.isEmpty) {
+        val header =
+          SummaryListRow(
+            key("containers"),
+            valueKey("site.no"),
```

Comment:
For consistency with other multi-rows sections, please use `site.none`

Reply:
This has been pulled from existing code so it would represent a content change, so I think unless stated it should stay the same? Do we want to ask Tim?

(see: https://github.com/hmrc/customs-declare-exports-frontend/blob/main/app/views/helpers/summary/ContainersHelper.scala#L59)